app/Jobs/ElasticSearchAliasInit.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Jobs;
+
+use Illuminate\Support\Facades\Log;
+use App\Http\Curl\HttpRequest;
+use App\WikiDb;
+
+class ElasticSearchAliasInit extends Job
+{
+    private $wikiId;
+    private $dbName;
+    private $sharedPrefix;
+
+    /**
+     * @param int $wikiId
+     * @param string $dbName
+     */
+    public function __construct( int $wikiId, string $sharedPrefix = null )
+    {
+        $this->wikiId = $wikiId;
+        $this->sharedPrefix = $sharedPrefix ?? getenv( 'ELASTICSEARCH_SHARED_INDEX' );
+    }
+
+    /**
+     * @param HttpRequest $request
+     */
+    public function handle( HttpRequest $request )
+    {
+        $timeStart = microtime( true );
+        Log::info( __METHOD__ . ": Updating Elasticsearch aliases for $this->wikiId" );
+
+        if ( !$this->sharedPrefix ) {
+            Log::error( __METHOD__ . ": Missing shared index prefix for $this->wikiId" );
+            $this->fail(
+                new \RuntimeException( "Missing shared index prefix for $this->wikiId" )
+            );
+            return;
+        }
+
+        Log::info( __METHOD__ . ": Using '$this->sharedPrefix' as the shared prefix for $this->wikiId" );
+
+        $this->dbName = WikiDb::where( 'wiki_id', $this->wikiId )->pluck( 'name' )->first();
+        if ( !$this->dbName ) {
+            Log::error( __METHOD__ . ": Failed to get database name for $this->wikiId" );
+            $this->fail(
+                new \RuntimeException( "Failed to get database name for $this->wikiId" )
+            );
+            return;
+        }
+
+        $actions = [];
+        foreach ( [ 'content', 'general' ] as $index ) {
+            $notAliasedIndex = $this->sharedPrefix . '_' . $index . '_first';
+            $filter = $this->dbName . '-';
+            $aliases = [
+                $this->dbName,
+                $this->dbName . '_' . $index,
+                $this->dbName . '_' . $index . '_first'
+            ];
+
+            foreach ( $aliases as $alias ) {
+                array_push( $actions, [
+                    'add' => [
+                        'index' => $notAliasedIndex,
+                        'alias' => $alias,
+                        'routing' => $alias,
+                        'filter' => [ 'prefix' => [ 'wiki' => $filter ] ]
+                    ]
+                ] );
+            }
+        }
+
+        $request->setOptions( [
+            CURLOPT_URL => getenv( 'ELASTICSEARCH_HOST' ) . '/_aliases',
```

Comment:
I fear we might get ourselves in a pickle here because as per https://github.com/wbstack/api/pull/773/files#diff-659bc1299066ca604751eb8a6212f15ed4e308b952dd6e8932cf03172b16a3aaR22 at time we set this to be a comma separated list of hosts. Maybe we should actually use a totally different variable here uniquely for this purpose which is making alias indexes? `ELASTICSEARCH_ALIAS_INDEX_CREATION_HOST`?